scripts/build-tokens.js

```diff
@@ -33,6 +33,20 @@ async function run() {
   fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/global-win32/reactnative/tokens-aliases.json'));
   fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/global-win32/reactnative/tokens-controls.json'));
 
+  console.log('Generating macOS global tokens...');
+  child_process.execSync(
+    'yarn transform-tokens --in ./packages/theming/theme-tokens/src/pipeline-input/token-input.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-brand.macos.json --out ./packages/theming/theme-tokens/src/global-macos --p reactnative',
+  );
+  fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/global-macos/reactnative/tokens-aliases.json'));
+  fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/global-macos/reactnative/tokens-controls.json'));
+
+  console.log('Generating macOS light mode tokens...');
```

Comment:
Are you skipping dark mode for now?

Reply:
forgot to push the latest commits, they're added in ef20d47cff4149350648dd84506fd113e1e1d78b